### PR TITLE
Addressing solhint warnings in solidity v5 sources

### PR DESCRIPTION
**Pull request by justinjmoses**

### CODE REVIEW

File: contracts/FeePoolEternalStorage.sol [hunk 1]
```diff
@@ -6,7 +6,7 @@ import "./LimitedSetup.sol";
 
 // https://docs.synthetix.io/contracts/FeePoolEternalStorage
 contract FeePoolEternalStorage is EternalStorage, LimitedSetup {
-    bytes32 constant LAST_FEE_WITHDRAWAL = "last_fee_withdrawal";
+    bytes32 public constant LAST_FEE_WITHDRAWAL = "last_fee_withdrawal";
```
Comment (hav-noms) [inline contracts/FeePoolEternalStorage.sol:9]: internal to not pollute the public ABI

File: contracts/EtherCollateral.sol [hunk 2]
```diff
@@ -18,13 +18,13 @@ contract EtherCollateral is Owned, Pausable, ReentrancyGuard, MixinResolver {
...
-    uint256 constant ONE_HUNDRED = 1e18 * 100;
+    uint256 internal constant ONE_THOUSAND = 1e18 * 1000;
+    uint256 internal constant ONE_HUNDRED = 1e18 * 100;
 
-    uint256 constant SECONDS_IN_A_YEAR = 31536000; // Common Year
+    uint256 internal constant SECONDS_IN_A_YEAR = 31536000; // Common Year
 
     // Where fees are pooled in sUSD.
-    address constant FEE_ADDRESS = 0xfeEFEEfeefEeFeefEEFEEfEeFeefEEFeeFEEFEeF;
+    address public constant FEE_ADDRESS = 0xfeEFEEfeefEeFeefEEFEEfEeFeefEEFeeFEEFEeF;
```
[the remark this answers is not in the document]
Comment (jacko125) [reply contracts/EtherCollateral.sol:27]: Do we want to have FEE_ADDRESS still public for people to see?

File: contracts/LimitedSetup.sol [hunk 3]
```diff
@@ -3,13 +3,13 @@ pragma solidity ^0.5.16;
 
 // https://docs.synthetix.io/contracts/LimitedSetup
 contract LimitedSetup {
-    uint setupExpiryTime;
+    uint internal setupExpiryTime;
```
Comment (jacko125) [inline contracts/LimitedSetup.sol:6]: public - might be helpful for this to be readable from abi if we need to check if LimitedSetup is expired.

File: contracts/EtherCollateral.sol (hunk 2 above)
[the remark this answers is not in the document]
Comment (justinjmoses) [reply contracts/EtherCollateral.sol:27]: They should really be looking it up via FeePool though

**Pull request merged**